examples/basic_usage/README.md

```diff
@@ -56,9 +56,9 @@ No requirements.
 
 | Name | Description |
 |------|-------------|
-| arn | The EC2 instance ARN |
-| availability\_zone | The AZ where the EC2 instance is deployed |
-| id | The EC2 instance ID |
-| private\_ip | The private IP of the EC2 instance |
+| arn | The EC2 instance ARN. |
+| availability\_zone | The AZ where the EC2 instance is deployed. |
+| id | The EC2 instance ID. |
+| private\_ip | The private IP of the EC2 instance. |
 | public\_ip | The public IP of the EC2 instance |
```

Comment:
Output description would need to be changed.

```suggestion
| public\_ip | The public IP of the EC2 instance. |
```

Reply:
See commit f0aecc6.